mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -170,6 +170,7 @@ def test_io_set_raw(fnames, tmpdir):
     shutil.copyfile(op.join(base_dir, 'test_raw.fdt'),
                     negative_latency_fname.replace('.set', '.fdt'))
     event_id = {eeg.event[0].type: 1}
+    # XXXX this warning is not done yet
```

Comment:
This relates to this:
https://github.com/mne-tools/mne-python/blob/672d66b9f3121ef3ba7f7aba1c6050094c56904c/mne/io/eeglab/eeglab.py#L687

and I don't really know what to do with it. 'cos we'll never have a latency of -1. since now we use as reference 'meas_date', at most it would become `raw.first_samp-1`